README.md

```diff
@@ -54,3 +54,59 @@ Cloudflare Client V4 API Email of the current user.
 to each request.
 
 5. Build an API Client for the Cloudflare Host API which adds the Host Key to all requests.
+
+## config.json 
+
+config.json has the following structure:
```

Comment:
Yeah but the ./config.json could sit exist if the user wants to over write our default values. I was assuming that is what this was referring to. I can remove it if you want. Let me know.